cmd/entrypoint/main.go

```diff
@@ -106,9 +116,6 @@ type RealPostWriter struct{}
 var _ entrypoint.PostWriter = (*RealPostWriter)(nil)
 
 func (*RealPostWriter) Write(file string) {
```

Comment:
why do we not need the check that no post file is passed in?

Reply:
Ah, good point, mistake on my part :sweat_smile: